_posts/2021-06-30-join-demodam-hackathon.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Join the Demodam hackathon"
+date: 2021-06-30
+author: Alba Roza
+type: blogpost
+excerpt: Information on how to join the following Demodam hackathon
+categories:
+  - Demodam
```

Comment:
Do we want a specific category for Demodam? I don't think we have one for any other codebase yet. I think we mostly use "news" instead.

Reply:
Changed!